source/reference/beta-program.txt

```diff
@@ -49,7 +52,8 @@ following disclaimer:
    migrations). Beta capabilities are not intended for use to migrate 
    production workloads.
 
-   Do you want to continue? (y/n):
+When you first run the {+c2c-beta-program-short+} binary, you must accept the 
+disclaimer again.
```

Comment:
nit: we can adjust "you must [...]" to be "you will be prompted to accept the disclaimer". I think the "again" can be confusing, since the mongosync binary has a command line prompt that requires the user to accept via the prompt, which would only happen once they have access to the binary